web/styles/dark_theme.css

```diff
@@ -643,6 +643,11 @@
         background-color: hsl(212deg 28% 8% / 75%);
     }
 
+    #admin_users_table .deactivated_user,
+    #admin_deactivated_users_table .reactivated_user {
+        color: hsl(236deg 33% 90%/50%);
```

Comment:
I am not sure how these colors are selected, but they are definitely not the ones used in the Notification settings panel.
Though this looks fine to me, I think would be better to get more feedback on it.

Reply:
I have changed the color now. Now the color used in Notification settings panel is used